docs/Reference/TypeScript.md

```diff
@@ -207,21 +207,23 @@ Here are some options on how to achieve this.
 
 Fastify offers two packages wrapping `json-schema-to-ts` and `typebox`:
 
-- `@fastify/type-provider-json-schema-to-ts`
-- `@fastify/type-provider-typebox`
+- `@fastify/type-provider-json-schema-to-ts` - [GitHub](https://github.com/fastify/fastify-type-provider-json-schema-to-ts)
```

Comment:
Just add the links to the list items rather than having a separate "GitHub" bit?

Reply:
Sure I can do that. But then I will also update the type provider doc page.

Reply:
@Fdawgs I updated the docs with your feedback and like I mentioned, I also updated the Type provider page to keep all links consistent.